### DISCUSSION TITLE
Welcome to The-Drag-and-Drop-Component-Suite-for-Delphi Discussions!

### DISCUSSION THREAD

**Discussion by landrix** [DelphiPraxis/The-Drag-and-Drop-Component-Suite-for-Delphi#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by srh-subramanian** [comment]
Hi Team, Firstly I would like to thank team for developing such a wonderful component. My clarification is when I use TDropComboTarget component, I'm not able to drag and drop the zip file from disk to Delphi Application. Could you please help to resolve my clarification or any workaround to sort this problem out
